src/pages/overview/document-generation-api/templatetags.md

```diff
@@ -146,12 +146,76 @@ the output document.
 
 ![Image placeholder replaced by the image data](../images/image_tag_final.png)
 
+## Conditionals
+
+Show or hide content in the document based on the conditions.
+
+**How to insert Conditional Tags**
+
+<Media slots="video"/>
+
+<https://images-tv.adobe.com/mpcv3/5158/671b4819-2e8c-44a1-8667-e9dfd2556133_1615916356.854x480at800_h264.mp4#t=3>
+
+
+<InlineAlert slots="text"/>
+
+The condition must be wrapped inside *expr()*. <br/> Supported
+conditional operators are =, !=, >=, >, <=, <.
+
+JSON representation of the input data:
+
+```json
+{
+  "state": "CA"
+}
+```
+
+
+**Conditional Sections**
+
+Show or hide paragraphs based on conditions.
+
+![Showing an paragraph based on a condition](../images/conditional_section.png)
+
+
+**Conditional Phrases**
+
+Show or hide text inside a paragraph based on conditions.
+
+![Showing a text inside a paragraph based on a condition](../images/conditional_phrase.png)
+
 ## Tables
+### Insert Tables using `<table>` HTML Element
+
+To insert tables in the document, add any text tag containing html table string as the value.
+
+Below is a json containing textTag with an HTML Table value.
+
+```json
+{
+  "Table_Data": "<table border='1' cellpadding='0' cellspacing='0' align='right' style='width:30%'>\n\t<tr>\n\t\t<th>Name</th>\n\t\t<th>Age</th>\n\t\t<th>Gender</th>\n\t</tr>\n\t<tr>\n\t\t<td>Sheldon</td>\n\t\t<td>29</td>\n\t\t<td>Male</td>\n\t</tr>\n\t<tr>\n\t\t<td>Leonard</td>\n\t\t<td>30</td>\n\t\t<td>Male</td>\n\t</tr>\n\t<tr>\n\t\t<td>Howard</td>\n\t\t<td>28</td>\n\t\t<td>Male</td>\n\t</tr>\n\t<tr>\n\t\t<td>Raj</td>\n\t\t<td>29</td>\n\t\t<td>Male</td>\n\t</tr>\n</table>\n"
+}
+```
+**Usage**
+
+Placeholder tag is replaced by the table generated using html table string provided in the placeholder tag value.
```

Comment:
change "using html table string" to "using html string" - the word table is used too much there.